content/en/profiler/enabling/ruby.md

```diff
@@ -136,7 +136,7 @@ Alternatively, you can set profiler parameters in code with these functions, ins
 | `c.profiling.enabled`                                 | Boolean | If set to `true`, enables the profiler. Defaults to `false`.                                                                            |
 | `c.profiling.allocation_enabled`                      | Boolean | Set to `true` to enable allocation profiling. It requires the profiler to be enabled already. Defaults to `false`.                      |
 | `c.profiling.advanced.experimental_heap_enabled`      | Boolean | Set to `true` to enable heap live objects profiling. It requires that allocation profiling is enabled as well. Defaults to `false`.     |
-| `c.profiling.advanced.experimental_heap_size_enabled` | Boolean | Set to `true` to enable heap live size profiling. It requires that heap live objects profiling is enabled as well. Defaults to `false`. |
+| `c.profiling.advanced.experimental_heap_size_enabled` | Boolean | Set to `true` to enable heap live size profiling. It requires that heap live objects profiling is enabled as well. Defaults to the same value as heap enabled. |
```

Comment:
```suggestion
| `c.profiling.advanced.experimental_heap_size_enabled` | Boolean | Set to `true` to enable heap live size profiling. It requires that heap live objects profiling is enabled as well. Defaults to the same value as `DD_PROFILING_EXPERIMENTAL_HEAP_ENABLED`. |
```

Reply:
For this one, should it be "Defaults to the same value as `experimental_heap_size_enabled`" or "Defaults to the same value as `c.profiling.advanced.experimental_heap_enabled`"?

My thinking is it's weird to mix set-via-code and set-via-environment-variable, at least in our docs (even though customers can and do mix them).

Reply:
I trust your judgment on this one, if you'd like to use your first suggestion. You know the landscape better than I do, just wanted to add a bit more clarity!

Reply:
Ack! Gonna go with:

```suggestion
| `c.profiling.advanced.experimental_heap_size_enabled` | Boolean | Set to `true` to enable heap live size profiling. It requires that heap live objects profiling is enabled as well. Defaults to the same value as `experimental_heap_size_enabled`. |
```